### PR TITLE
Update files included with embedded distributions

### PR DESCRIPTION
**Pull request by labkey-tchad**
#### Rationale
The service scripts and `application.properties` need some polish before we ship them.
`server/configs/application.properties` is used for dev deployments and TeamCity. This file should have a comprehensive list of available properties.
`server/configs/webapps/embedded/config/application.properties` is the one included in distributions. This should have a more pared down set of properties and won't have the `@@value@@` placeholders used for string replacement by the gradle plugin.

#### Related Pull Requests
* https://github.com/LabKey/server/pull/752

#### Changes
* Tidy up `application.properties` files
* Remove some internal properties from properties included with distributions (CSP configs and json access log)
* Remove mentions of Tomcat from service scripts

### CODE REVIEW

File: server/configs/application.properties [hunk 1]
```diff
@@ -59,11 +74,32 @@ mail.smtpUser=@@smtpUser@@
...
+#context.resources.ldap.ConfigFactory.factory=org.labkey.premium.ldap.LdapConnectionConfigFactory
+#context.resources.ldap.ConfigFactory.host=myldap.mydomain.com
+#context.resources.ldap.ConfigFactory.port=389
+#context.resources.ldap.ConfigFactory.principal=cn=read_user
+#context.resources.ldap.ConfigFactory.credentials=read_user_password
+#context.resources.ldap.ConfigFactory.useTls=false
+#context.resources.ldap.ConfigFactory.useSsl=false
+#context.resources.ldap.ConfigFactory.sslProtocol=SSLv3
+
+#useLocalBuild#spring.devtools.restart.additional-paths=@@pathToServer@@/build/deploy/modules,@@pathToServer@@/build/deploy/embedded/config
```
Comment (labkey-susanh) [inline server/configs/application.properties]: This line (recently introduced) looks like it got dropped:

`#useLocalBuild#spring.devtools.restart.trigger-file=.restartTrigger`

Comment (labkey-tchad) [reply server/configs/application.properties]: Indeed it was. Thank you

File: server/configs/application.properties [hunk 2]
```diff
@@ -35,21 +33,38 @@ context.resources.jdbc.labkeyDataSource.validationQuery=SELECT 1
 #context.resources.jdbc.labkeyDataSource.logQueries=true
 #context.resources.jdbc.labkeyDataSource.displayName=Alternate Display Name
 
+## Add additional external data sources with a prefix of context.resources.jdbc.<dataSourceName>.
+## At a minimum, they must include the required properties; driverClassName, url, username, and password
```
Comment (labkey-jeckels) [inline server/configs/application.properties]:
```suggestion
## At a minimum, they must include the required properties: driverClassName, url, username, and password
```

File: server/configs/webapps/embedded/config/application.properties [hunk 3]
```diff
@@ -31,14 +29,18 @@ context.resources.jdbc.labkeyDataSource.validationQuery=SELECT 1
 #context.resources.jdbc.labkeyDataSource.logQueries=true
 #context.resources.jdbc.labkeyDataSource.displayName=Alternate Display Name
 
-#context.resources.jdbc.@@extraJdbcDataSource@@.driverClassName=@@extraJdbcDriverClassName@@
-#context.resources.jdbc.@@extraJdbcDataSource@@.url=@@extraJdbcUrl@@
-#context.resources.jdbc.@@extraJdbcDataSource@@.username=@@extraJdbcUsername@@
-#context.resources.jdbc.@@extraJdbcDataSource@@.password=@@extraJdbcPassword@@
+## Add additional external data sources with a prefix of context.resources.jdbc.<dataSourceName>.
+## At a minimum, they must include the required properties; driverClassName, url, username, and password
```
Comment (labkey-jeckels) [inline server/configs/webapps/embedded/config/application.properties]:
```suggestion
## At a minimum, they must include the required properties: driverClassName, url, username, and password
```

File: server/configs/webapps/embedded/config/application.properties [hunk 4]
```diff
@@ -31,14 +29,18 @@ context.resources.jdbc.labkeyDataSource.validationQuery=SELECT 1
...
+#context.resources.jdbc.extraDataSource.url=jdbc:postgresql://localhost:5432/extraDataSource
+#context.resources.jdbc.extraDataSource.username=<username>
+#context.resources.jdbc.extraDataSource.password=<password>
 
-context.encryptionKey=@@encryptionKey@@
+context.encryptionKey=<encryptionKey>
+#context.oldEncryptionKey=
 
-## By default, we deploy to the root context path. However, some servers have historically used /labkey or even /cpas
+## By default, we serve LabKey at the root context path (e.g. http://localhost:8080)
```
Comment (labkey-jeckels) [inline server/configs/webapps/embedded/config/application.properties]:
```suggestion
## We strongly recommend deploying LabKey at the root context path (e.g. http://localhost:8080)
```

**Review by labkey-jeckels — APPROVED** [review]

**Review by labkey-jony — APPROVED** [review]

File: server/configs/webapps/embedded/labkey_server.service
Comment (labkey-jony) [inline server/configs/webapps/embedded/labkey_server.service:1]: I tried this revised version in a recent test, but the variables do not appear to be respected.

For example, `$LABKEY_HOME` literally gets treated as the name of the folder, not as a substitution for `/labkey/labkey`. This causes a directory to be created called `'$LABKEY_HOME'` with the single-quotes with a path of `/labkey/labkey/'$LABKEY_HOME'`

In my original example file, I had to hardcode these paths for them to work as well.

https://gist.github.com/labkey-jony/12d8242e906947fa46a7b62f3238a522

**Pull request merged**